ci-operator/config/quarkus-qe/quarkus-test-suite/quarkus-qe-quarkus-test-suite-main__quarkus-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,49 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .openshift-interop/
+  to: quarkus-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tests:
+- as: quarkus-interop-aws
+  cron: 0 6 * * 1
+  remote_api: true
```

Comment:
This line is no longer needed. The api is now added by default on periodic jobs.